src/Runner.Common/JobServerQueue.cs

```diff
@@ -703,17 +703,17 @@ private List<TimelineRecord> MergeTimelineRecords(List<TimelineRecord> timelineR
                     timelineRecord.State = rec.State ?? timelineRecord.State;
                     timelineRecord.WorkerName = rec.WorkerName ?? timelineRecord.WorkerName;
 
-                    if (rec.ErrorCount != null && rec.ErrorCount > 0)
+                    if (rec.ErrorCount > 0)
                     {
                         timelineRecord.ErrorCount = rec.ErrorCount;
                     }
```

Comment:
`ErrorCount`, `WarningCount`, and `NoticeCount` are no longer nullable (previously `Int32?`, now `Int32`)